### PR TITLE
feature/PEER-609: Add navigation buttons

### PR DESCRIPTION
**Pull request by alexeishulga**

### CODE REVIEW

File: app/components/Documentation/EditDocumentation.tsx [hunk 1]
```diff
@@ -68,10 +75,26 @@ const EditDocumentation: React.FC<EditDocumentationProps> = ({
...
   const refOverlay = useRef<HTMLDivElement>(null);
   const [paddingLeft, setPaddingLeft] = useState<number>(86);
 
+  const treeArray = getFlatDataFromTree({
+    treeData: documentationMenu?.map((node) => ({ ...node })),
+    getKey: (node) => node.id,
+    getNodeKey: ({ treeIndex }) => treeIndex,
+    ignoreCollapsed: false,
+  });
+  const currentArrayIndex = treeArray.filter(
```
Comment (Lampardks) [inline app/components/Documentation/EditDocumentation.tsx:89]: use method find

File: app/components/Documentation/EditDocumentation.tsx [hunk 2]
```diff
@@ -68,10 +75,26 @@ const EditDocumentation: React.FC<EditDocumentationProps> = ({
...
+    (item) => item?.node.id == viewArticleId,
+  )[0]?.treeIndex;
+  const sliceTitle = (title: string): string =>
+    title?.length < 26 ? title : title?.substr(0, 25) + '...';
```
Comment (Lampardks) [inline app/components/Documentation/EditDocumentation.tsx:93]: don't use magic numbers, use constants

File: app/components/Documentation/EditDocumentation.tsx [hunk 3]
```diff
@@ -68,10 +75,26 @@ const EditDocumentation: React.FC<EditDocumentationProps> = ({
...
+
+  const isStartArticle = currentArrayIndex < 1;
```
Comment (Lampardks) [inline app/components/Documentation/EditDocumentation.tsx:95]: don't use magic numbers, use constants

File: app/components/Documentation/EditDocumentation.tsx [hunk 4]
```diff
@@ -68,10 +75,26 @@ const EditDocumentation: React.FC<EditDocumentationProps> = ({
...
+  const isLastArticle = currentArrayIndex == treeArray.length - 1;
```
Comment (Lampardks) [inline app/components/Documentation/EditDocumentation.tsx:96]: ===

File: app/components/Documentation/EditDocumentation.tsx [hunk 5]
```diff
@@ -137,6 +160,47 @@ const EditDocumentation: React.FC<EditDocumentationProps> = ({
...
+      setViewArticleDispatch(id);
+      setEditArticleDispatch({
+        id: id,
+        parentId: '1',
+        isEditArticle: false,
+      });
+    }
+  };
+  const onClickPrevArticle = (): void => {
+    const prevArrayId = treeArray.filter(
```
Comment (Lampardks) [inline app/components/Documentation/EditDocumentation.tsx:177]: use method find

File: app/components/Documentation/EditDocumentation.tsx [hunk 6]
```diff
@@ -137,6 +160,47 @@ const EditDocumentation: React.FC<EditDocumentationProps> = ({
...
+      (item) => item?.treeIndex == currentArrayIndex - 1,
+    )[0]?.node.id;
+    onClickArticle(prevArrayId);
+  };
+
+  const onClickNextArticle = (): void => {
+    const nextArrayId = treeArray.filter(
```
Comment (Lampardks) [inline app/components/Documentation/EditDocumentation.tsx:184]: use method find

combine functions onClickNextArticle  and onClickPrevArticle  into one, use a closure

```
const onClickArticle = (typeButton) => () => {
  ...
}
```

File: app/components/Documentation/EditDocumentation.tsx [hunk 7]
```diff
@@ -137,6 +160,47 @@ const EditDocumentation: React.FC<EditDocumentationProps> = ({
...
+      (item) => item?.treeIndex == currentArrayIndex + 1,
+    )[0]?.node.id;
+    onClickArticle(nextArrayId);
+  };
+
+  const onClickNextTitle = (): string => {
```
Comment (Lampardks) [inline app/components/Documentation/EditDocumentation.tsx:190]: combine functions onClickNextTitle and onClickPrevTitle into one, use a closure

File: app/components/Documentation/EditDocumentation.tsx [hunk 8]
```diff
@@ -137,6 +160,47 @@ const EditDocumentation: React.FC<EditDocumentationProps> = ({
...
+    const nextTitle = treeArray.filter(
```
Comment (Lampardks) [inline app/components/Documentation/EditDocumentation.tsx:191]: use find

File: app/components/Documentation/EditDocumentation.tsx [hunk 9]
```diff
@@ -137,6 +160,47 @@ const EditDocumentation: React.FC<EditDocumentationProps> = ({
...
+      (item) => item?.treeIndex == currentArrayIndex + 1,
+    )[0]?.node.title;
+    return sliceTitle(nextTitle);
+  };
+
+  const onClickPrevTitle = (): string => {
+    const prevTitle = treeArray.filter(
```
Comment (Lampardks) [inline app/components/Documentation/EditDocumentation.tsx:198]: use find

File: app/components/Documentation/EditDocumentation.tsx [hunk 10]
```diff
@@ -229,6 +293,78 @@ const EditDocumentation: React.FC<EditDocumentationProps> = ({
                 )}
               </>
             )}
+            <div
```
Comment (Lampardks) [inline app/components/Documentation/EditDocumentation.tsx:296]: move all pagination to a separate component

File: app/components/Documentation/components/ViewContent.tsx [hunk 11]
```diff
@@ -16,33 +31,157 @@ const styled = {
...
+  const currentArticleId = documentationArticle?.id;
+  const currentArrayIndex = treeArray.filter(
+    (item) => item?.node.id == currentArticleId,
+  )[0]?.treeIndex;
+  const sliceTitle = (title: string): string =>
+    title?.length < 26 ? title : title?.substr(0, 25) + '...';
+  const isStartArticle = currentArrayIndex < 1;
+  const isLastArticle = currentArrayIndex == treeArray.length - 1;
+
+  const onClickPrevArticle = (): void => {
```
Comment (Lampardks) [inline app/components/Documentation/components/ViewContent.tsx:59]: you used the same functions in another component, this is a duplication, find a way to merge

File: app/components/Documentation/EditDocumentation.tsx (hunk 1 above)
Comment (alexeishulga) [reply app/components/Documentation/EditDocumentation.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1255/commits/c018592c251a49f7d793813c5d02d03715e687c9

File: app/components/Documentation/EditDocumentation.tsx (hunk 3 above)
Comment (alexeishulga) [reply app/components/Documentation/EditDocumentation.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1255/commits/c018592c251a49f7d793813c5d02d03715e687c9

File: app/components/Documentation/EditDocumentation.tsx (hunk 4 above)
Comment (alexeishulga) [reply app/components/Documentation/EditDocumentation.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1255/commits/c018592c251a49f7d793813c5d02d03715e687c9

File: app/components/Documentation/EditDocumentation.tsx (hunk 5 above)
Comment (alexeishulga) [reply app/components/Documentation/EditDocumentation.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1255/commits/c018592c251a49f7d793813c5d02d03715e687c9

File: app/components/Documentation/EditDocumentation.tsx (hunk 7 above)
Comment (alexeishulga) [reply app/components/Documentation/EditDocumentation.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1255/commits/c018592c251a49f7d793813c5d02d03715e687c9

File: app/components/Documentation/EditDocumentation.tsx (hunk 8 above)
Comment (alexeishulga) [reply app/components/Documentation/EditDocumentation.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1255/commits/c018592c251a49f7d793813c5d02d03715e687c9

File: app/components/Documentation/EditDocumentation.tsx (hunk 9 above)
Comment (alexeishulga) [reply app/components/Documentation/EditDocumentation.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1255/commits/c018592c251a49f7d793813c5d02d03715e687c9

File: app/components/Documentation/EditDocumentation.tsx (hunk 10 above)
Comment (alexeishulga) [reply app/components/Documentation/EditDocumentation.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1255/commits/c018592c251a49f7d793813c5d02d03715e687c9

File: app/components/Documentation/components/ViewContent.tsx (hunk 11 above)
Comment (alexeishulga) [reply app/components/Documentation/components/ViewContent.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1255/commits/c018592c251a49f7d793813c5d02d03715e687c9

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx [hunk 12]
```diff
@@ -0,0 +1,109 @@
...
+  const isDesktop = useMediaQuery('(min-width: 577px)');
+  const treeArray = getDataFromTree(documentationMenu);
+
+  const currentArrayIndex = getcurrentArrayIndex(treeArray, id);
+
+  const isStartArticle = currentArrayIndex < 1;
+  const isLastArticle = currentArrayIndex === treeArray.length - 1;
+  const TITLE_LENGTH = isDesktop ? 26 : 50;
+  const TITLE_LAST_INDEX = isDesktop ? 25 : 49;
+  const TITLE_START_INDEX = 0;
```
Comment (Lampardks) [inline app/components/Documentation/components/Pagination/PaginationDocumentation.tsx:32]: transfer to a separate file with constants and import

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx [hunk 13]
```diff
@@ -0,0 +1,109 @@
...
+    const ipfsHash = getIpfsHashFromBytes32(arrayId);
+    createdHistory.push(routes.documentation(ipfsHash, arrayTitle));
+  };
+
+  const getcurrentArrayTitle = (typeButton: 'prev' | 'next') => {
+    const stepDirection = typeButton === PREV_TYPE_BUTTON ? -1 : +1;
+    const title = treeArray?.find(
+      (item) => item?.treeIndex === currentArrayIndex + stepDirection,
+    )?.node.title;
+    return title?.length < TITLE_LENGTH
```
Comment (Lampardks) [inline app/components/Documentation/components/Pagination/PaginationDocumentation.tsx:54]: why? why don't you use css for shortening?

File: app/components/Documentation/components/Pagination/ButtonPaginationDesktop.tsx [hunk 14]
```diff
@@ -0,0 +1,52 @@
...
+import { useTranslation } from 'react-i18next';
+import { css } from '@emotion/react';
+import ArrowDownIcon from 'icons/ArrowDown';
+import { ButtonPaginationProps } from '../../types';
+
+const ButtonPaginationDesktop: React.FC<ButtonPaginationProps> = ({
+  next,
+  onClickPaginationArticle,
+  getcurrentArrayTitle,
+  NEXT_TYPE_BUTTON,
```
Comment (Lampardks) [inline app/components/Documentation/components/Pagination/ButtonPaginationDesktop.tsx:11]: not props, use import

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx [hunk 15]
```diff
@@ -0,0 +1,109 @@
...
+                onClickPaginationArticle
+              }
+              getcurrentArrayTitle={getcurrentArrayTitle}
+              NEXT_TYPE_BUTTON={NEXT_TYPE_BUTTON}
+              PREV_TYPE_BUTTON={PREV_TYPE_BUTTON}
+            />
+          </div>
+          <div className={`${isLastArticle ? 'dn' : null}`}>
+            <ButtonPaginationDesktop
+              next
```
Comment (Lampardks) [inline app/components/Documentation/components/Pagination/PaginationDocumentation.tsx:81]: delete

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx [hunk 16]
```diff
@@ -0,0 +1,109 @@
...
+            isStartArticle ? 'jcfe' : isLastArticle ? 'jcfs' : 'jcc'
+          } full-width mb16 mt32 fww`}
+        >
+          <div className={`${isStartArticle ? 'dn' : null}`}>
+            <ButtonPaginationDesktop
+              currentArrayIndex={currentArrayIndex}
+              treeArray={treeArray}
+              onClickPaginationArticle={
+                onClickPaginationArticleEditDocumentation ||
+                onClickPaginationArticle
```
Comment (Lampardks) [inline app/components/Documentation/components/Pagination/PaginationDocumentation.tsx:72]: onClickPaginationArticle(PREV_TYPE_BUTTON)

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx [hunk 17]
```diff
@@ -0,0 +1,109 @@
...
+            />
+          </div>
+          <div className={`${isLastArticle ? 'dn' : null}`}>
+            <ButtonPaginationDesktop
+              next
+              currentArrayIndex={currentArrayIndex}
+              treeArray={treeArray}
+              onClickPaginationArticle={
+                onClickPaginationArticleEditDocumentation ||
+                onClickPaginationArticle
```
Comment (Lampardks) [inline app/components/Documentation/components/Pagination/PaginationDocumentation.tsx:86]: onClickPaginationArticle(NEXT_TYPE_BUTTON)

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx [hunk 18]
```diff
@@ -0,0 +1,109 @@
...
+  const isStartArticle = currentArrayIndex < 1;
+  const isLastArticle = currentArrayIndex === treeArray.length - 1;
+  const TITLE_LENGTH = isDesktop ? 26 : 50;
+  const TITLE_LAST_INDEX = isDesktop ? 25 : 49;
+  const TITLE_START_INDEX = 0;
+  const TITLE_END = '...';
+  const NEXT_TYPE_BUTTON = 'next';
+  const PREV_TYPE_BUTTON = 'prev';
+
+  const onClickPaginationArticle = (typeButton: 'prev' | 'next') => () => {
```
Comment (Lampardks) [inline app/components/Documentation/components/Pagination/PaginationDocumentation.tsx:37]:
```
const onClickPaginationArticle = (typeButton: PREV_TYPE_BUTTON | NEXT_TYPE_BUTTON) => () => {
    const stepDirection = typeButton === PREV_TYPE_BUTTON ? -1 : +1;
    const arrayId = treeArray.find(
      (item) => item?.treeIndex === currentArrayIndex + stepDirection,
    )?.node.id;

    if (typeof onClickPaginationArticleEditDocumentation === 'function') {
      onClickPaginationArticleEditDocumentation(arrayId);
      return;
    }

    const arrayTitle = treeArray.find(
      (item) => item?.treeIndex === currentArrayIndex + stepDirection,
    )?.node.title;
    const ipfsHash = getIpfsHashFromBytes32(arrayId);

    createdHistory.push(routes.documentation(ipfsHash, arrayTitle));
  };
```

File: app/components/Documentation/EditDocumentation.tsx [hunk 19]
```diff
@@ -142,6 +150,28 @@ const EditDocumentation: React.FC<EditDocumentationProps> = ({
...
+      setViewArticleDispatch(id);
+      setEditArticleDispatch({
+        id: id,
+        parentId: '1',
+        isEditArticle: false,
+      });
+    }
+  };
+
+  const onClickPaginationArticle = (typeButton: 'prev' | 'next') => () => {
```
Comment (Lampardks) [inline app/components/Documentation/EditDocumentation.tsx:167]:
```
const onClickPaginationArticle = (arrayId) => {
    onClickArticle(arrayId);
  };
```

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx [hunk 20]
```diff
@@ -0,0 +1,109 @@
...
+          <div className={`${isStartArticle ? 'dn' : null}`}>
+            <ButtonPaginationDesktop
+              currentArrayIndex={currentArrayIndex}
+              treeArray={treeArray}
+              onClickPaginationArticle={
+                onClickPaginationArticleEditDocumentation ||
+                onClickPaginationArticle
+              }
+              getcurrentArrayTitle={getcurrentArrayTitle}
+              NEXT_TYPE_BUTTON={NEXT_TYPE_BUTTON}
```
Comment (Lampardks) [inline app/components/Documentation/components/Pagination/PaginationDocumentation.tsx:75]: delete

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx [hunk 21]
```diff
@@ -0,0 +1,109 @@
...
+            <ButtonPaginationDesktop
+              next
+              currentArrayIndex={currentArrayIndex}
+              treeArray={treeArray}
+              onClickPaginationArticle={
+                onClickPaginationArticleEditDocumentation ||
+                onClickPaginationArticle
+              }
+              getcurrentArrayTitle={getcurrentArrayTitle}
+              NEXT_TYPE_BUTTON={NEXT_TYPE_BUTTON}
```
Comment (Lampardks) [inline app/components/Documentation/components/Pagination/PaginationDocumentation.tsx:89]: delete

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx [hunk 22]
```diff
@@ -0,0 +1,109 @@
...
+              PREV_TYPE_BUTTON={PREV_TYPE_BUTTON}
```
Comment (Lampardks) [inline app/components/Documentation/components/Pagination/PaginationDocumentation.tsx:90]: delete

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx [hunk 23]
```diff
@@ -0,0 +1,109 @@
...
+            />
+          </div>
+        </div>
+      ) : (
+        <ButtonPaginationMobile
+          currentArrayIndex={currentArrayIndex}
+          treeArray={treeArray}
+          onClickPaginationArticle={onClickPaginationArticle}
+          getcurrentArrayTitle={getcurrentArrayTitle}
+          NEXT_TYPE_BUTTON={NEXT_TYPE_BUTTON}
```
Comment (Lampardks) [inline app/components/Documentation/components/Pagination/PaginationDocumentation.tsx:100]: delete

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx [hunk 24]
```diff
@@ -0,0 +1,109 @@
...
+          PREV_TYPE_BUTTON={PREV_TYPE_BUTTON}
```
Comment (Lampardks) [inline app/components/Documentation/components/Pagination/PaginationDocumentation.tsx:101]: delete

Comment (alexeishulga) [reply app/components/Documentation/components/Pagination/PaginationDocumentation.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1255/commits/8366eac25ae6411cc2701193402f2a07047418c5

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx (hunk 23 above)
Comment (alexeishulga) [reply app/components/Documentation/components/Pagination/PaginationDocumentation.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1255/commits/8366eac25ae6411cc2701193402f2a07047418c5

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx (hunk 22 above)
Comment (alexeishulga) [reply app/components/Documentation/components/Pagination/PaginationDocumentation.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1255/commits/8366eac25ae6411cc2701193402f2a07047418c5

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx (hunk 21 above)
Comment (alexeishulga) [reply app/components/Documentation/components/Pagination/PaginationDocumentation.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1255/commits/8366eac25ae6411cc2701193402f2a07047418c5

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx [hunk 25]
```diff
@@ -0,0 +1,109 @@
...
+            <ButtonPaginationDesktop
+              currentArrayIndex={currentArrayIndex}
+              treeArray={treeArray}
+              onClickPaginationArticle={
+                onClickPaginationArticleEditDocumentation ||
+                onClickPaginationArticle
+              }
+              getcurrentArrayTitle={getcurrentArrayTitle}
+              NEXT_TYPE_BUTTON={NEXT_TYPE_BUTTON}
+              PREV_TYPE_BUTTON={PREV_TYPE_BUTTON}
```
[the remark this answers is not in the document]
Comment (alexeishulga) [reply app/components/Documentation/components/Pagination/PaginationDocumentation.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1255/commits/8366eac25ae6411cc2701193402f2a07047418c5

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx (hunk 20 above)
Comment (alexeishulga) [reply app/components/Documentation/components/Pagination/PaginationDocumentation.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1255/commits/8366eac25ae6411cc2701193402f2a07047418c5

File: app/components/Documentation/EditDocumentation.tsx (hunk 19 above)
Comment (alexeishulga) [reply app/components/Documentation/EditDocumentation.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1255/commits/8366eac25ae6411cc2701193402f2a07047418c5

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx (hunk 18 above)
Comment (alexeishulga) [reply app/components/Documentation/components/Pagination/PaginationDocumentation.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1255/commits/8366eac25ae6411cc2701193402f2a07047418c5

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx (hunk 17 above)
Comment (alexeishulga) [reply app/components/Documentation/components/Pagination/PaginationDocumentation.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1255/commits/8366eac25ae6411cc2701193402f2a07047418c5

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx (hunk 16 above)
Comment (alexeishulga) [reply app/components/Documentation/components/Pagination/PaginationDocumentation.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1255/commits/8366eac25ae6411cc2701193402f2a07047418c5

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx (hunk 15 above)
Comment (alexeishulga) [reply app/components/Documentation/components/Pagination/PaginationDocumentation.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1255/commits/8366eac25ae6411cc2701193402f2a07047418c5

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx (hunk 12 above)
Comment (alexeishulga) [reply app/components/Documentation/components/Pagination/PaginationDocumentation.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1255/commits/303a47c06288eb7a7876b1135fe75d5894b3359f

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx (hunk 13 above)
Comment (alexeishulga) [reply app/components/Documentation/components/Pagination/PaginationDocumentation.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1255/commits/303a47c06288eb7a7876b1135fe75d5894b3359f

File: app/components/Documentation/components/Pagination/ButtonPaginationDesktop.tsx (hunk 14 above)
Comment (alexeishulga) [reply app/components/Documentation/components/Pagination/ButtonPaginationDesktop.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1255/commits/303a47c06288eb7a7876b1135fe75d5894b3359f

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx [hunk 26]
```diff
@@ -0,0 +1,94 @@
...
+  ) => {
+    const stepDirection = typeButton === PREV_TYPE_BUTTON ? -1 : +1;
+    const id = treeArray.find(
+      (item) => item?.treeIndex === currentArrayIndex + stepDirection,
+    )?.node.id;
+    const title = treeArray?.find(
+      (item) => item?.treeIndex === currentArrayIndex + stepDirection,
+    )?.node.title;
+    return [id, title];
+  };
```
Comment (Lampardks) [inline app/components/Documentation/components/Pagination/PaginationDocumentation.tsx:42]:
```
const getTitleButton = (
    typeButton: typeof PREV_TYPE_BUTTON | typeof NEXT_TYPE_BUTTON,
  ) => {
    const stepDirection = typeButton === PREV_TYPE_BUTTON ? -1 : +1;
    
    const title = treeArray?.find(
      (item) => item?.treeIndex === currentArrayIndex + stepDirection,
    )?.node.title;
    return title;
  };
```

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx [hunk 27]
```diff
@@ -0,0 +1,94 @@
...
+            isStartArticle ? 'jcfe' : isLastArticle ? 'jcfs' : 'jcc'
+          } full-width mb16 mt32 fww`}
+        >
+          <div className={`${isStartArticle ? 'dn' : null}`}>
+            <ButtonPaginationDesktop
+              typeButton={PREV_TYPE_BUTTON}
+              onClickPaginationArticle={onClickPaginationArticle(
+                PREV_TYPE_BUTTON,
+              )}
+              getcurrentArrayTitle={getButtonContent(PREV_TYPE_BUTTON)[1]}
```
Comment (Lampardks) [inline app/components/Documentation/components/Pagination/PaginationDocumentation.tsx:70]:
```
title={getTitleButton(PREV_TYPE_BUTTON)}
```

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx [hunk 28]
```diff
@@ -0,0 +1,94 @@
...
+            />
+          </div>
+          <div className={`${isLastArticle ? 'dn' : null}`}>
+            <ButtonPaginationDesktop
+              typeButton={NEXT_TYPE_BUTTON}
+              onClickPaginationArticle={onClickPaginationArticle(
+                NEXT_TYPE_BUTTON,
+              )}
+              getcurrentArrayTitle={getButtonContent(NEXT_TYPE_BUTTON)[1]}
```
Comment (Lampardks) [inline app/components/Documentation/components/Pagination/PaginationDocumentation.tsx:79]:
```
title={getTitleButton(NEXT_TYPE_BUTTON)}
```

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx [hunk 29]
```diff
@@ -0,0 +1,94 @@
...
+            />
+          </div>
+        </div>
+      ) : (
+        <ButtonPaginationMobile
+          onClickPaginationArticle={onClickPaginationArticle}
+          getcurrentArrayTitle={getButtonContent}
```
Comment (Lampardks) [inline app/components/Documentation/components/Pagination/PaginationDocumentation.tsx:86]:
```
getTitleButton={getTitleButton}
```

Comment (alexeishulga) [reply app/components/Documentation/components/Pagination/PaginationDocumentation.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1255/commits/1f72634ed34165196f0727c7c0056236611f1b85

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx (hunk 28 above)
Comment (alexeishulga) [reply app/components/Documentation/components/Pagination/PaginationDocumentation.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1255/commits/1f72634ed34165196f0727c7c0056236611f1b85

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx (hunk 27 above)
Comment (alexeishulga) [reply app/components/Documentation/components/Pagination/PaginationDocumentation.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1255/commits/1f72634ed34165196f0727c7c0056236611f1b85

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx (hunk 26 above)
Comment (alexeishulga) [reply app/components/Documentation/components/Pagination/PaginationDocumentation.tsx:38]: done: https://github.com/peeranha/peeranha-web/pull/1255/commits/155bec5c9eb03b83400caf912c5f244167ddd56a

**Review by Lampardks — APPROVED** [review]

**Pull request closed without merging**